spring-boot-autoconfigure/src/main/java/org/axonframework/springboot/autoconfig/AxonServerActuatorAutoConfiguration.java

```diff
@@ -40,11 +43,12 @@
 @ConditionalOnProperty(name = "axon.axonserver.enabled", matchIfMissing = true)
 public class AxonServerActuatorAutoConfiguration {
 
+    @ConditionalOnMissingBean(AxonServerHealthIndicator.class)
     @Bean
     public AxonServerHealthIndicator axonServerHealthIndicator(AxonServerConnectionManager connectionManager) {
         return new AxonServerHealthIndicator(connectionManager);
     }
-
+    @ConditionalOnMissingBean(SimpleStatusAggregator.class)
```

Comment:
Ah, now this raises a valuable point. I'd wager an environment should be able to have N amount of `SimpleStatusAggregators`. 
Wherein each aggregator provides some additional info to the overall status.

If we make the `AxonServerStatusAggregator` conditional based on a `SimpleStatusAggregator`, I'd guess the chances are high it's replaced in favor of an aggregator for something entirely different.

Reply:
Thanks for bringing this up, I was tempted to set the whole PR to WIP because of this but forgot to do so.
My thought behind using `SimpleStatusAggregator` as the condition-bean, was that I supposed the `SimpleStatusAggregator` aggregates the "overall" status of the instance and therefore will be unique.

If that assumption is wrong, I happily change it to `AxonServerStatusAggregator`

Reply:
After some investigation, we figured out it's expected that there's a single `StatusAggregator` instance per health endpoint.
As such, overriding the existing one may be fair, but if end-users also do this, their variant should take control.
As such, the way you've implemented it is completely fine :+1: